### PR TITLE
RT Auth: use config for RT login/password

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

**Comment by acoffman** [comment]
+1

File: etc/genome/spec/rt_auth.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# The 'rt_auth' configuration variable holds the login password
+# to connect to RT @ MGI.
```
Comment (tmooney) [inline etc/genome/spec/rt_auth.yaml]: Are these configuration variables really MGI-specific?

Comment (ebelter) [inline etc/genome/spec/rt_auth.yaml]: Nope! I can remove that comment.

**Comment by tmooney** [comment]
+1, assuming that the change to add the values to the `config.yaml` will be coming!

**Comment by ebelter** [comment]
@tom  What would our team be without trust :smile:

**Comment by ebelter** [comment]
The new ENVs that were implemented in puppet break the CLE validation snapshot. I have added defaults values to allow this PR to be merged.

@tmooney @acoffman Please review.

**Comment by tmooney** [comment]
Could we add the new values to the `config.yaml` but *not* create `GENOME_*` environment variables (in order to not break CLE)?  Those environment variables don't do anything, since nothing in any snapshot looks for them.

**Comment by ebelter** [comment]
Review welcome: https://github.com/genome/puppet-modules/pull/4

**Comment by ebelter** [comment]
Puppet changes merged and applied. Merging.

**Pull request merged**